client/dashboard/task-list/style.scss

```diff
@@ -255,3 +255,36 @@
 		margin-bottom: $gap-smallest;
 	}
 }
+
+.woocommerce-task-card__prompt {
```

Comment:
Can we add the little arrow/pointer here?

<img width="268" alt="Screen Shot 2019-10-10 at 10 36 13 AM" src="https://user-images.githubusercontent.com/10561050/66534801-fd84f580-eb49-11e9-886f-fc9465ebbe2f.png">

Reply:
Added in https://github.com/woocommerce/woocommerce-admin/pull/3021/commits/50b40260cb5c93bdfcb532d6ae891f26ab663d32.

<img width="997" alt="Screen Shot 2019-10-10 at 10 35 20 AM" src="https://user-images.githubusercontent.com/689165/66579078-37ee9280-eb4a-11e9-9654-ac1c4d5d90ac.png">